src/hpguppi_ata_blade_obs_rawdisk_thread.c

```diff
@@ -273,12 +273,18 @@ static void *run(hashpipe_thread_args_t * args)
           }
           else{
```

Comment:
L268 above needs to not assume a single beam (ie `nbeams = 0`), then the primary if statement is `if nbeams > 0`... L276 below should still be `nbeams = 1` because that is the case when nbeams is present (>0) but not correct, so we still want a -beam suffix on the filename...